proposals/NNNN-change-registration-manager.md

```diff
@@ -0,0 +1,63 @@
+# change-registration-manager
+
+* Proposal: [SDL-0052](NNNN-change-registration-manager.md)
+* Author: [Kujtim Shala](https://www.github.com/kshala)
+* Status: **Awaiting review**
+* Impacted Platforms: [iOS]
+
+## Introduction
+
+ChangeRegistration RPC allows an app to manipulate parameters that were send. Including the app name, TTS name and voice synonyms of the app. This proposal aims to make it more convenient for the app developer and to provide an API of a similar style unlike the RPC request.
```

Comment:
This paragraph's grammar could be improved; it is somewhat difficult to understand. Is it building an API similar to RPCs?